func.yaml

```diff
@@ -1,3 +1,8 @@
+schema_version: 20180708
```

Comment:
You can delete these files, they are the result of `fn init` in your working directory.